lib/StatusIndicator/styles.scss

```diff
@@ -56,6 +50,12 @@ $status-indicator-horizontal-margin: .5rem !default;
   }
 }
 
+.status-indicator--bordered {
+  border: 2px solid $color-border-base;
+  border-radius: .25rem;
+  padding: 0.7rem;
```

Comment:
Is there any way we can use a standardised value here? Spacing, font sizes and colours all should use settings where possible.

If you NEED to override consistent global settings then this can be done by adding this style to the components config area (the area at the top of this file with the section comment "config") and then overriding it in the consuming app.